### PR TITLE
Refactor expenses

### PR DESCRIPTION
**Pull request by itsmattli**
Refactoring, migrations, models to match Eloquent model. Added jquery to js/include/. Refactored some code in DashboardController to return branch to working order.

### CODE REVIEW

File: database/migrations/2017_03_14_041508_UpdateUsersTable.php [hunk 1]
```diff
@@ -15,6 +15,7 @@ public function up()
     {
         Schema::table('users', function (Blueprint $table) {
             // Primary Key
+            $table->primary('username');
```
Comment (nikoootine) [inline database/migrations/2017_03_14_041508_UpdateUsersTable.php]: I thought we're making user id as our primary key now?

File: public/js/include/jquery-3.2.0.min.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/*! jQuery v3.2.0 | (c) JS Foundation and other contributors | jquery.org/license */
```
Comment (nikoootine) [inline public/js/include/jquery-3.2.0.min.js]: jquery is included in laravel by default. see app.js

File: database/migrations/2017_03_13_211740_CreateFriendsTable.php [hunk 3]
```diff
@@ -14,6 +14,7 @@ class CreateFriendsTable extends Migration
     public function up()
     {
         Schema::create('friends', function (Blueprint $table) {
+            $table->increments('id');
```
Comment (nikoootine) [inline database/migrations/2017_03_13_211740_CreateFriendsTable.php]: if we're adding id as primary key, we should now make the combination of username1 and username2 unique too

File: database/migrations/2017_03_14_041508_UpdateUsersTable.php (hunk 1 above)
Comment (itsmattli) [inline database/migrations/2017_03_14_041508_UpdateUsersTable.php]: i only made the change for table with composite primary keys

**Pull request merged**